lib/improver/tests/utilities/cube_manipulation/test_merge_cubes.py

```diff
@@ -89,6 +91,21 @@ def setUp(self):
         self.prob_enuk.attributes.update({'mosg__grid_domain': 'uk_extended'})
         self.prob_enuk.attributes.update({'mosg__grid_version': '1.2.0'})
 
+        # Setup two non-Met Office model example configuration cubes.
+        # Using a simple temperature data array, one cube set is setup
+        # as a deterministic model, the other as an ensemble.
+        self.data = (
+            np.linspace(275.0, 284.0, 12).reshape(3, 4).astype(np.float32))
+        self.data_3d = np.array([self.data, self.data, self.data])
+
+        self.cube_non_mo_det = set_up_variable_cube(self.data_3d)
```

Comment:
This will create a 3D cube with a realization coordinate - hence the undesirable behaviour in the test below.  You want `self.cube_non_mo_det = set_up_variable_cube(self.data)`.

Reply:
Fixed, thanks!